apps/prairielearn/src/batched-migrations/20250318180548_course_instance_usages__workspaces__backfill_v3.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from 'zod';
+
+import { makeBatchedMigration } from '@prairielearn/migrations';
+import { loadSqlEquiv, queryAsync, queryRow } from '@prairielearn/postgres';
+
+const sql = loadSqlEquiv(import.meta.url);
+
+const END_DATE = '2025-03-21T00:00:00Z';
```

Comment:
We'll actually backfill all workspace data, not just the data that was incorrectly generated before #https://github.com/PrairieLearn/PrairieLearn/pull/11590. This is because our original backfill incorrectly assumed that rows in a date range were equivalent to rows in an ID range. This is almost true for submissions and grading jobs, which are inserted in order, but it's not true for workspaces, which can be restarted. To fix this we'll just re-backfill workspaces completely. These are very fast to backfill in any case.